teens_ui/vue.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  publicPath: '/teens',
```

Comment:
adding publicpath, index.html can import webpacked files such as js and css in subdirectory